components/ArtPiecesPreview.js

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ArtPiecesPreview({ piece }) {
+  const { imageSource: image, name: title, artist } = piece;
+
+  const StyledContainer = styled.div`
+    position: relative;
+    height: 300px;
+  `;
+
+  return (
+    <>
+      <figure>
+        <StyledContainer>
+          <Image
+            src={image}
+            alt={`This is the image ${title} by ${artist}.`}
+            fill
+            style={{ objectFit: "contain" }}
+          />
+        </StyledContainer>
+        <figcaption>{`"${title}" by ${artist}`}</figcaption>
+      </figure>
+    </>
```

Comment:
The fragments `<> </>`  aren't necessary because the component already has a single parent element `<figure>`.  Removing the fragments will simplify the code without affecting the functionality.
```suggestion
      <figure>
        <StyledContainer>
          <Image
            src={image}
            alt={`This is the image ${title} by ${artist}.`}
            fill
            style={{ objectFit: "contain" }}
          />
        </StyledContainer>
        <figcaption>{`"${title}" by ${artist}`}</figcaption>
      </figure>
```